### DISCUSSION TITLE
Newly installed repository not showing up in App developer portal

### DISCUSSION THREAD

**Discussion by arthurgeek** [renovatebot/renovate#23092] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

I had renovate working [on this repo](https://github.com/arthurgeek/homelab). Today, I created a [new repo](https://github.com/arthurgeek/vaultwarden-fly-template) and renovate dashboard/onboarding never kicked off. So, I decided to uninstalll renovate on my GitHub and install again. Now, when I go to [Mend's developer portal](https://developer.mend.io) I see: "There are no GitHub organizations connected to this account that have Mend Developer Platform installed. To install Mend Developer Platform go by this link to Github marketplace." even though it is already installed.

### What you tried so far.

- Uninstalling and installing the Github app.
- Deleting my Mend account

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
I see that https://github.com/arthurgeek/vaultwarden-fly-template/issues/1 was created:

<img width="472" alt="image" src="https://github.com/renovatebot/renovate/assets/6311784/6fe920bd-a102-4afe-8ca2-9de5b462c956">

Looks like it was about 25 minutes later than your post:

<img width="361" alt="image" src="https://github.com/renovatebot/renovate/assets/6311784/1dd6ef2c-0bc6-463b-a70f-2982d3746e36">

One of the team will look into why the delay's happening. When you log into the portal now do you see the repos listed?

**Reply by arthurgeek** [reply]
Yes, I can see both repos in the portal now. Thanks!

Now I have another unrelated issue. Can you help me understand why the ARG-dependencies are not picked up by renovate? https://github.com/arthurgeek/vaultwarden-fly-template/blob/main/Dockerfile#L1-L5

**Reply by rarkins** [reply]
Please create separate discussions per topic, and also aim to have more descriptive titles than you chose for this one (they are unhelpful for people who may search in future)

**Reply by arthurgeek** [reply]
Thanks, will do. I had it phrased way better, but then had to turn this into a question, and couldn't figure out a good one.

**Comment by rarkins** [comment]
@arthurgeek can you give more information about the onboarding problem you had?

You mention that you _created_ https://github.com/arthurgeek/vaultwarden-fly-template however you have _Selected_ repos not _All_ repos configured, right? So at which point did you configure Renovate for that repo and how did you do that?

**Reply by arthurgeek** [reply]
After pushing everything, I then went to Renovate's App Page, clicked configure and selected the new repo. That was minutes after first push.

**Reply by rarkins** [reply, marked as answer]
Thanks, I think we've identified the problem. Around midnight-1:30am UTC each day we do a periodic sync of every installation to catch any repos we missed during the previous day. If someone adds a repo during that time it goes to the _back_ of that queue so may take up to 1.5 hours before it's sync'd, leading you to think it's missed/lost.

The queue was drained around 2am so it went right away that time

**Reply by arthurgeek** [reply]
Thanks for looking into it!
